src/Native/Runtime/inc/rhbinder.h

```diff
@@ -667,12 +667,8 @@ enum RhEHClauseKind
     RH_EH_CLAUSE_FAULT              = 1,
 
     // local exceptions
-    RH_EH_CLAUSE_METHOD_BOUNDARY    = 2,    // /eh:rh
-    RH_EH_CLAUSE_FAIL_FAST          = 3,    // /eh:rh
-
-    // CLR exceptions
-    RH_EH_CLAUSE_FILTER             = 2,    // /eh:clr
-    RH_EH_CLAUSE_UNUSED             = 3,    // /eh:clr
+    RH_EH_CLAUSE_METHOD_BOUNDARY    = 2,
+    RH_EH_CLAUSE_FAIL_FAST          = 3
```

Comment:
Did you keep the wrong set?  I would have expected you to keep RH_EH_CLAUSE_FILTER and that's it?